create_infotexts.py

```diff
@@ -360,33 +365,34 @@ def process_depicted_place(self, place_string, places_mapping):
             return
 
         if place_string in places_mapping:
-            if not places_mapping[place_string]["wikidata"] == "-":
-                 place_as_wikitext = "{{city|1={wikidata}}}".format(
-                    wikidata=places_mapping[place_string]["wikidata"][2:]
-                )
-            elif place_string == "Stockholm":  # Mainly interiors from buildings gardens
-                self.meta_cats.append("[[Media_contributed_by_SMVK_taken_in_Stockholm]]")
-                self.data["depicted_place"] = ""
+            if place_string == "Stockholm":  # Mainly interiors from buildings gardens
+                # print("Stockholm in places_mapping{}".format(place_string))
+                self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
+                self.data["depicted_place"] = "Stockholm"
 
             elif place_string == "Macheras":  # No WP article, highly ambiguous. Might refer to "Machairas Monestary"
+                self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
                 self.meta_cats.append("Media_contributed_by_SMVK_possibly_depicting_Machairas_Monastary")
-                self.data["depicted_place"] = ""
-
+                self.data["depicted_place"] = "Macheras"
 
-            elif not places_mapping[place_string]["commons"] == "-":
-                place_as_wikitext = "{{city|1={commons}}".format(
-                    commons=places_mapping[place_string]["commons"]
+            elif places_mapping[place_string]["wikidata"] != "-":
+                place_as_wikitext = "{{{{city|1={wikidata}}}}}".format(
+                    wikidata=places_mapping[place_string]["wikidata"]
                 )
 
             else:
-                self.meta_cats.append("Media_contributed_by_SMVK_without_depicted_place_value")
-                self.data["depicted_place"] = ""
-                return
+                self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
+                self.data["depicted_place"] = place_string
+        else:
+            self.meta_cats.append("Media_contributed_by_SMVK_without_depicted_place_value")
+            place_as_wikitext = place_string
 
         # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
         if "commons" in places_mapping[place_string].keys() and places_mapping[place_string]["commons"] != "-":
```

Comment:
Argh. That is me getting confused between the person json (where we changed common to commoncat) and the places which we scrape. Sorry for that.
Still the important bit (moving the statement) needed to be done and I believe commonscat is a beter name anyway.